bps/bps_ApPipe.yaml

```diff
@@ -10,41 +10,37 @@ campaign: smallrun_example
 
 submitPath: ${PWD}/bps/{outCollection}
 computeSite: ncsapool
-# Memory allocated for each quantum, in MBs
+# The default BPS walltime is on the order of 3 days.
+# That's much more than the vast majority of jobs need,
+# and it can cause your jobs not to run if you're too close to a maintenance window.
+# Set the default walltime as appropriate for what you are running (12 hours below).
+site:
+  ncsapool:
+    profile:
+      condor:
+        +Walltime: 43200
+# Memory allocated for each quantum, in MBs; can be overridden on a per-task basis.
+# The bps default is 2048 MB, the same as this example, but it's kept here as a reminder.
 requestMemory: 2048
-# CPUs to use per quantum
-requestCpus: 1
 
-# Example arguments, like the ones you would send to pipetask run from the command line
 payload:
-  runInit: true
   # UPDATE THIS to also set the output collection name
   payloadName: DM-XXXXX_ApPipe_example
   # UPDATE THIS to point to the correct repository
   butlerConfig: /repo/main/butler.yaml
   # UPDATE THIS and be sure it includes collections with raws, calibs, refcats, skymaps, and templates
   inCollection: HSC/defaults,'u/${USER}/templates'
-  # Note: keep timestamp in outCollection so you don't get a zillion sub-runs
-  output : 'u/${USER}/{payloadName}'
-  outCollection: '{output}/{timestamp}'
+  # UPDATE THIS as it is the place you want to store output.
+  output: 'u/{operator}/{payloadName}'
```

Comment:
A bit odd to have the not-otherwise-used-or-defined `operator` here yet `${USER}` a couple lines above for `inCollection`.

Reply:
Switching to {operator} and adding a sentence saying it defaults to the value of USER.